### DISCUSSION TITLE
Schema dumping is not supported when using SQLite

### DISCUSSION THREAD

**Discussion by IsmailAshour** [laravel/framework#34259] [General]
I use SQLite for testing purposes. it's lightweight and in-memory db.

SQLite dumping is not supported, so schema:dump will not be useful.

Are there any ideas?

**Comment by driesvints** [comment]
The docs were incorrect on this. It's only available for PostgreSQL and MySQL. But we're accepting prs to implement it on Sqlite.

**Comment by paras-malhotra** [comment]
Just submitted a PR to support this: https://github.com/laravel/framework/pull/34323
